examples/websearch/websearch-tools.ts

```diff
@@ -0,0 +1,149 @@
+import ollama, { Ollama } from 'ollama'
+import type { Message } from 'ollama'
+
+async function main() {
+  const API_KEY = process.env.OLLAMA_API_KEY || ''
+  const MODEL = process.env.OLLAMA_MODEL || 'gpt-oss'
+
+  const client = new Ollama({
+    headers: { Authorization: `Bearer ${API_KEY}` },
```

Comment:
just collapse the loading of the key directly in here - make sure to test the case of what happens if it is unset